frontend/src/e2e-playwright/specs/6_mobile/child-attendances.spec.ts

```diff
@@ -193,7 +193,50 @@ describe('Child mobile attendances', () => {
   })
 })
 
+const assertAttendanceCounts = async (
+  coming: number,
+  present: number,
+  departed: number,
+  absent: number,
+  total: number
+) =>
+  await waitUntilEqual(() => listPage.getAttendanceCounts(), {
+    coming,
+    present,
+    departed,
+    absent,
+    total
```

Comment:
Come to think of it, total could be omitted from params and just summed here `total: coming + present + departed + absent`